packages/kamino-sdk/src/Kamino.ts

```diff
@@ -1704,23 +1700,13 @@ export class Kamino {
 
     let tokenAMinPostDepositBalance = userTokenBalances.a?.sub(amountToDeposit);
 
-    let swapper: SwapperIxBuilder = swapIxsBuilder
```

Comment:
make the swapIxsBuilder non optional on every single function, i hate these insidious and hidden dependencies, basically footguns